lib/import_js/importer.rb

```diff
@@ -130,6 +145,16 @@ def message(str)
       @editor.message("ImportJS: #{str}")
     end
 
+    # @param numbers [Set]
+    # @param range [Range]
+    # @return [Boolean]
+    def any_numbers_within_range?(numbers, range)
+      numbers.each do |number|
+        return true if range.include?(number)
+      end
+      false
+    end
```

Comment:
I originally had it that way, but I think I was running into weirdness with not being able to return from the block, since that will return from the function. I suppose I could have written this as a `find` instead of a `each`, which would have helped. Feel free to rework.